examples/standalone_panorama/outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "username" {
+  description = "Initial administrative username to use for VM-Series."
+  value       = var.vmseries_username
+}
+
+output "password" {
+  description = "Initial administrative password to use for VM-Series."
+  value       = local.vmseries_password
+  sensitive   = true
+}
+
+output "mgmt_ip" {
```

Comment:
```suggestion
output "mgmt_ips" {
```

Reply:
there is some inconsistency in the outputs. For examples I would say we should stick to plural. 90% of outputs are maps or lists. In this case it sounds more natural to make it plural. For resources that are single in nature (like `password` or `username`) - stick to singular.